ical.NET/Serialization/iCalendar/Serializers/DataTypes/GeographicLocationSerializer.cs

```diff
@@ -45,9 +46,9 @@ public GeographicLocation Deserialize(string value)
 
             double lat;
             double lon;
-            double.TryParse(values[0], out lat);
-            double.TryParse(values[1], out lon);
-            g.Latitude = lat;
+			double.TryParse(values[0], NumberStyles.Any, CultureInfo.InvariantCulture, out lat);
```

Comment:
It's because they're tabs (`\t`) instead of spaces. ical.net uses 4 spaces for each tab stop, not real tabs. When I bumped the nuget version number, I converted them to spaces.

It would be nice is Visual Studio had a "Figure out what this project is using, and copy the it" option for whitespace.